src/redux/post/post.actions.ts

```diff
@@ -34,13 +34,11 @@ export const newComment = createAsyncThunk(
   "posts/newComment",
   async ({ text, postId }: Comment, thunkApi) => {
     const state = thunkApi.getState() as RootState;
```

Comment:
Didn't I already mention this multiple times? ... Please use the typescript generics, don't use the assertion here.

Reply:
No state defenition seems to be required... when I remove RootState completely, the program functions as expected.